tests/OmniSharp.Roslyn.CSharp.Tests/FindReferencesFacts.cs

```diff
@@ -421,29 +421,34 @@ public class F$$oo {
         [Fact]
         public async Task MappedLocationFileNameProperlyRooted()
         {
-            var relativeFile = ".\\pages\\test.xaml";
+            var folderPath = Directory.GetCurrentDirectory();
+            var relativeFile = ".\\Pages\\Index.cshtml";
+            var mappedFilePath = Path.GetFullPath(Path.Combine(folderPath, relativeFile));
 
             var testFiles = new[]
             {
-                new TestFile("a.cs",
-                 @"public class Bar : F$$oo {}"),
-                new TestFile("b.cs", $@"
-                    #line 23 ""{relativeFile}""
-                    public class Foo {{ }}
+                new TestFile("Constants.cs",
+                 @"
+public static class Constants
+{
+    public const string My$$Text = ""Hello World"";
+}"),
+                new TestFile("index_cshtml.generated.cs", $@"
+                    #line 1 ""{relativeFile}""
+                    Constants.MyText
                     #line default
-                    #line hidden")
+                    #line hidden"),
+                new TestFile(mappedFilePath,
```

Comment:
Adding the mapped file as an additional document will create a valid DocumentId that an be found by file path but does not exist in the Project.Documents.